.github/workflows/test.yml

```diff
@@ -48,6 +48,8 @@ jobs:
         run: npm install --prefix ./.ci
       - name: Run integration tests
         run: ./.ci/integrationTest.ts
+        env:
+          BASE_ADDRESS: http://localhost:5000
```

Comment:
It's because I like to define them as closest to the place it is used as possible. 

Or in other words: the script should do as much as possible. 

Otherwise there would be several problems, e.g. it wouldn't be set if we ran the script locally.